Automattic/wp-calypso
# Checkout: Remove unused google vouchers code

#### Changes proposed in this Pull Request

Half a year ago, in #44133 we removed the Google Voucher cards from My Plans and checkout thank you pages. This PR removes the remaining code that was no longer used but was just lying around.

#### Testing instructions

* Smoke test My Plans page and purchasing a plan, verify everything works the same way as in production.
* Verify all tests still pass.
